src/sentry/api/endpoints/relay/register_challenge.py

```diff
@@ -50,7 +50,7 @@ def post(self, request: Request) -> Response:
 
         public_key = json_data.get("public_key")
         if not public_key:
-            return Response({"detail": "Missing public key"}, status=status.HTTP_400_FORBIDDEN)
+            return Response({"detail": "Missing public key"}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Why this change?

Reply:
Sorry, are you changing this to return a 400 instead of a 500?

Reply:
In order to satisfy mypy, I can either do:
```py
return Response({"detail": "Missing public key"}, status=status.HTTP_400_BAD_REQUEST)
```
which would fix the bug and start returning a 400, or 
```py
return Response({"detail": "Missing public key"}, status=status.HTTP_400_FORBIDDEN)  #type: ignore
```
which will leave the bug in place and keep returning 500s.

Reply:
My question isn't so much that your typing is wrong. I'm asking more, are there going to be any side effects of returning a 400 here instead of a 500? I don't know what this endpoint is used for.

Reply:
this is fine, let's do it

Reply:
I think the intent is to send a `403 Forbidden`  response here, see:
https://github.com/getsentry/relay/blob/2e924639d7bcfa24db69ba2ed78a82e2c07478e1/relay-server/src/actors/upstream.rs#L86-L94

Reply:
Double-checked, `400 Bad Request` is fine in this case. It's the same as if we failed to parse JSON.